pyproject.toml

```diff
@@ -19,17 +19,13 @@ esdbclient = {version = ">=1.0b3", optional = true}
 django = {version = ">=4,<6", optional = true}
```

Comment:
Build fails because poetry.lock wasn't regenerated. Would you mind locking it? Output from the job says how it should be doing

Reply:
done